tests/integration/modules/molecule/pipe_handler/molecule.yml

```diff
@@ -0,0 +1,10 @@
+---
+scenario:
+  name: pipe_handler
+platforms:
```

Comment:
I think we ought to test against three platforms (5.12.0, 5.13.1 and latest), please see how @tadeboro modified it for asset module

Reply:
oh yeah, totally forgot about that